lib/fog/compute/kubevirt/requests/get_livevm.rb

```diff
@@ -5,37 +5,55 @@ module Compute
     class Kubevirt
       class Real
         def get_livevm(name)
-          Livevm.parse object_to_hash( kubevirt_client.get_virtual_machine(name, 'default') )
+          Livevm.parse object_to_hash( kubevirt_client.get_virtual_machine(name, @namespace) )
```

Comment:
I would consider mentioning where @namesapce is defined.

Reply:
this is how we deal with objects in fog but I will add comment for clarity.